src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -88,14 +102,27 @@ export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
         }
     }
 
+    toggleModal = (panelName: string, genes: GenePanelToGene[]) => {
+        this.modalOpen = !this.modalOpen;
+        if (!this.modalOpen) {
+            return;
+        }
+        this.modalPanelName = panelName;
+        this.modalPanelGenes = genes;
+    }
+
+    closeModal = () => {
```

Comment:
same here.  Make this an @action decorated method.  @autobind will handle bindind to instance.